### ISSUE TITLE
Three options activated by default

### ISSUE THREAD

**Issue by Gabrielxd195** [LMMS/lmms#5053]
I would like some options to be activated by default. such as:

- Activate the oscilloscope by default.
- Name of the notes on the piano roll.
- HQ mode for output devices.

![Kazam_screenshot_00000](https://user-images.githubusercontent.com/26127693/59957546-516ab200-9467-11e9-8d4f-c9a46405fbed.png)

These options are helpful to users: notes on piano roll and oscilloscope. that's why I would like (if there are no problems) that these options come activated by default. but are disabled by default by '' saving '' resources or to '' improve performance '', do not you think they should be in the section '' Performance Configuration '' ?, I think it would be more intuitive if they are in that section.

**Comment by Spekular** [comment]
I agree with these except for the note labels. For me they're just clutter,
I expect many users know the keys on a piano and would feel the same.

**Comment by Gabrielxd195** [comment]
> I agree with these except for the note labels. For me they're just clutter, I expect many users know the keys on a piano and would feel the same.

You're right friend, this marked notes is more for a person who knows the musical notes; but remember that most users of LMMS are beginners, any detail however small that facilitates the workflow, is a point in favor of LMMS, since this program is very versatile and comfortable.

**Comment by Spekular** [comment]
I don't believe we should target beginners at the expense of more
experienced users though, that would cement LMMS's standing as a DAW "for
beginners" (implied: not for "real" producers). With the incredible
progress in 1.2 and master branch, I think LMMS has a great opportunity to
grow out of that label.

**Comment by Gabrielxd195** [comment]
> I don't believe we should target beginners at the expense of more experienced users though, that would cement LMMS's standing as a DAW "for beginners" (implied: not for "real" producers). With the incredible progress in 1.2 and master branch, I think LMMS has a great opportunity to grow out of that label.

You're right, but the idea is not to accommodate LMMS to beginners but to implement small tools and options that together improve the workflow for both beginners and more experienced professionals. I also do not think that a simple option of "label of the note" changes the concept of LMMS. if we have seen enough changes in stable and master what makes LMMS become a real DAW.

But if you want people to see what LMMS really is, let's start changing the concept of: '' LMMS is a free cross-platform alternative to commercial programs like FL Studio® ''.

**Comment by NETMANSKY** [comment]
@Spekular, @Gabrielxd195 I cant to name LMMS as sandbox and newbies tool. There is too difficult oriented for newbie in sound producing world without normally working FX presets base. I mean FX only and not default Instrument/FX preset chains.
